mix.exs

```diff
@@ -5,12 +5,12 @@ defmodule ProgressBar.Mixfile do
     [
       app: :progress_bar,
       version: "1.6.1",
-      elixir: "~> 1.0",
+      elixir: "~> 1.3",
```

Comment:
I guess that if phoenix and ecto both depend on elixir 1.3 to work on current versions, we could be a little restrictive for a `1.7.0` release. :wink:

Reply:
Hah, yeah, this seems reasonable. I'm not scared of bumping major, so we could even make it 2.0. Maybe along with a breaking `suffix:` change.

Sorry about the nondeterministic specs of #18 fame. I re-ran the build and now it passes…

Reply:
No problem at all! :)